server/config.py

```diff
@@ -71,3 +71,6 @@
 QUEUE_POP_DESIRED_PLAYERS = int(os.getenv('QUEUE_POP_DESIRED_PLAYERS', 8))
 # How many previous queue sizes to consider
 QUEUE_POP_TIME_MOVING_AVG_SIZE = int(os.getenv('QUEUE_POP_TIME_MOVING_AVG_SIZE', 5))
+
+#team id that players in FFA mode are assigned to
+FFA_TEAM = int(os.getenv('FFA_TEAM', 1))
```

Comment:
I'm not sure this should be a config variable. I think this is embedded in the ForgedAlliance executable somewhere so we couldn't change it even if we wanted to. I refactored it to a named constant though because it used to make some of the game outcome logic very hard to read.

Reply:
I moved it because I wanted to import it into other modules rather than duplicating the convention, so it had to go somewhere where it wouldn't create circular dependencies.
Would `FFA_TEAM = 1` without the `os.getenv` make sense?

Reply:
yea, I would just define it as a constant, without adding the option for it to be changed via the environment